internal/fvalidate/verapdf_validator.go

```diff
@@ -82,3 +83,32 @@ func (v *veraPDFValidator) Validate(path string) (string, error) {
 		return "", errors.New("PDF/A validation failed with an application error")
 	}
 }
+
+func RunCommand(name string, args ...string) (string, error) {
+	result := exec.Command(name, "--version") // #nosec: G204 -- trusted path.
+
+	output, err := result.Output()
+	if err != nil {
+		return "", err
+	}
+
+	return string(output), nil
+}
+
+var Run = RunCommand
+
+func (v *veraPDFValidator) Version() (string, error) {
```

Comment:
I think it would be good to add this to the validator interface (and its mock):

- Add it here: https://github.com/artefactual-sdps/preprocessing-sfa/blob/main/internal/fvalidate/fvalidate.go#L10
- Run `make gen-mock` to update the fake implementation